stable/fluent-bit/templates/config.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-config
+  labels:
+    app: {{ template "fullname" . }}
+  namespace: {{ .Values.namespace }}
```

Comment:
This should be left out of the yamls and passed to `helm install --namespace foo` on the CLI.